### PR TITLE
Dynamic binary size test comment message based on size change.

### PR DESCRIPTION
**Pull request by purple4reina**
### What does this PR do?

Update the Serverless Binary Size check to not say "Warning" when there is nothing to worry about.

### Motivation

Confusion.

### CODE REVIEW

File: test/integration/serverless_perf/write_message.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<summary>Debug info</summary>
+
+If you have questions, we are happy to help, come visit us in the [#serverless](https://dd.slack.com/archives/CBWDFKWV8) slack channel and provide a link to this comment.
+
+These dependencies were added to the serverless extension by this pull request:
+
+\`\`\`
+${VAR_DEPS}
+\`\`\`
+
```
Comment (GustavoCaso) [inline test/integration/serverless_perf/write_message.sh:25]: Should we ensure that we only show these lines if there are added dependencies?

File: test/integration/serverless_perf/write_message.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+filename="${HOME}/comment.md"
+echo "filename=$filename" >> "$GITHUB_OUTPUT"
+
+if [ "${VAR_COLD_START}" != 0 ]; then
```
Comment (GustavoCaso) [inline test/integration/serverless_perf/write_message.sh:6]: What are the values that `VAR_COLD_START` usually store?

**Review by DylanLovesCoffee — APPROVED** [review]

File: test/integration/serverless_perf/write_message.sh (hunk 1 above)
Comment (purple4reina) [reply test/integration/serverless_perf/write_message.sh:25]: This is a great idea now that we can build this message dynamically!

File: test/integration/serverless_perf/write_message.sh (hunk 2 above)
Comment (purple4reina) [reply test/integration/serverless_perf/write_message.sh:6]: The value is defined [here](https://github.com/DataDog/datadog-agent/blob/rey.abolofia/bin-size-info/.github/workflows/serverless-binary-size.yml#L77-L79)

```
          OUTPUT=$(( $OUTPUT / 100000 ))
          echo "cold start time changed by $OUTPUT ms"
          echo "coldstart=$OUTPUT" >> $GITHUB_OUTPUT
```

Therefore, the value is always a positive integer.

For example:

```bash
$ echo $(( 1 / 10 ))
0
$ echo $(( 9 / 10 ))
0
$ echo $(( 10 / 10 ))
1
$ echo $(( 20 / 10 ))
2
```

**Pull request merged**